buildenv/jenkins/JenkinsfileBase

```diff
@@ -471,14 +471,11 @@ def setup() {
 
 def getJobProperties() {
 	def jobProperties = "./aqa-tests/job.properties"
-	if (fileExists("${jobProperties}")) {
-		echo "readProperties file: ${jobProperties}"
-		def properties = readProperties file: "${jobProperties}"
-		if (properties.TEST_JDK_HOME) {
-			env.TEST_JDK_HOME = properties.TEST_JDK_HOME
-			echo "Reset TEST_JDK_HOME to ${TEST_JDK_HOME}"
-		}
-	}
+	echo "readProperties file: ${jobProperties}"
+	def properties = readProperties file: "${jobProperties}"
+	env.TEST_JDK_HOME = properties.TEST_JDK_HOME
+	echo "Reset TEST_JDK_HOME to ${TEST_JDK_HOME}"
+
```

Comment:
I think we still need both if statements here. In else, we can print the clear message and fail the build.